views/index.pug

```diff
@@ -1,12 +1,18 @@
 extends layout
 
 block content
-  h1= title
-  p Welcome to #{title}
+  h1= DUMO
   div.newToDo
-    form(action = 'createProjectRoute' method = 'post')
-      input(placeholder = 'Enter new Project')
-  //- div.project.list
-  //-   ul
-  //-     each project in projects
-  //-       li=project.project_name
+    form(action = '/createProject' method = 'post')
```

Comment:
Keep consistent with whitespace before and after `=`. Parts of your code have whitespace surrounding the equality operator and other parts don't surround it with whitespace. Go through your code and change these inconsistencies to stick to one standard.